### DISCUSSION TITLE
Storage location of Demo habitat assets.

### DISCUSSION THREAD

**Discussion by JonathanTownsendForReal** [SimDynamX/SpaceTeamsPRO#19] [2023 Mars City Design Competition Q&A] [answered]
I am attempting to find the model files for the habitats used in the demo in order to scale our models. Does anyone know where they are stored or what their scale is?

**Comment by connorjak** [comment, marked as answer]
Here is a copy of `SpaceTeamsPro_0.20.7\Compute_Server\packages\STUniversity\ReferenceSimulation.json`:
[ReferenceSimulation.json](https://github.com/SimDynamX/SpaceTeamsPro/files/15102010/ReferenceSimulation.json)

In this version of our mars reference sim, we do some things that are only possible for Space Teams core developers to set up the habitat as one entity. You can copy the full entity parameters from the sim config above if you want to compare the AlphaHabitat to your mars city.

```json
    {
        "#Required": {
          "Name": "AlphaHabitat",
          "Type": "BP",
          "Systems": [ "SnapAssetToPlanet", "ParamWrangler" ]
        },
        "#BP": {
          "BP_Path": "/Game/StaticMeshes/MarsCityHabs/Habitat_1_All_In_One/AlphaHab.AlphaHab_C"
        },
        ...
```

I recommend using the VRIK skeleton (in `/VRIK`) as a comparison tool to see if the size of your 3D models make sense:

![image](https://github.com/SimDynamX/SpaceTeamsPro/assets/19719887/9c09cc86-7ae2-418e-a969-90f01b8471ef)
